cluster.go

```diff
@@ -33,6 +33,8 @@ type ClusterOptions struct {
 	ReadOnly bool
 	// Allows routing read-only commands to the closest master or slave node.
 	RouteByLatency bool
+	// Allows balanced routing of read-only commands to master and slave nodes.
+	LoadBalanceNodes bool
```

Comment:
Please change this to:

```
// Allows routing read-only commands to the random master or slave node.
RouteRandomly bool
```

So it matches `RouteByLatency` more closely.

Reply:
done in 22d7121